packages/react-test-renderer/package.json

```diff
@@ -22,6 +22,7 @@
     "object-assign": "^4.1.1",
     "prop-types": "^15.6.2",
     "react-is": "^16.8.6",
+    "react-shallow-renderer": "^16.12.0",
```

Comment:
A bit unfortunate we didn't start with 1.0.0 or similar. So that you could break it more often if necessary. Although I guess nothing prevents you from jumping to 17 later without waiting for us.